src/main/java/org/broadinstitute/hellbender/engine/VariantWalkerBase.java

```diff
@@ -42,7 +41,7 @@ void initializeFeatures() {
 
         //Note: we override this method because we don't want to set feature manager to null if there are no FeatureInputs.
         //This is because we have at least 1 source of features (namely the driving dataset).
-        features = new FeatureManager(this, FEATURE_CACHE_LOOKAHEAD, cloudPrefetchBuffer, cloudIndexPrefetchBuffer,
+        features = new FeatureManager(this, FeatureDataSource.DEFAULT_QUERY_LOOKAHEAD_BASES, cloudPrefetchBuffer, cloudIndexPrefetchBuffer,
```

Comment:
Add an explicit comment pointing out that you're using a different lookahead value for side inputs.